src/traces/scatter/attributes.js

```diff
@@ -398,9 +398,67 @@ module.exports = {
         description: [
             'Sets the fill color.',
             'Defaults to a half-transparent variant of the line color,',
-            'marker color, or marker line color, whichever is available.'
+            'marker color, or marker line color, whichever is available.',
+            'If fillgradient is specified, fillcolor is ignored except for',
+            'setting the background color of the hover label, if any.'
         ].join(' ')
     },
+    fillgradient: extendFlat({
+        orientation: {
+            valType: 'enumerated',
+            values: ['radial', 'horizontal', 'vertical', 'none'],
```

Comment:
For other `orientation` attributes we have *h* and *v* options instead of *horizontal* and *vertical*.
Do we really need *none* option? It could be left *undefined*. 
I suggest we have the following options:
```js
values: ['r', 'h', 'v'],
```
and don't set the `dflt`.